doc/network.tex

```diff
@@ -257,39 +258,38 @@ \subsubsection{Network connectivity}\label{network-connectivity}
 
 \paragraph{Participate as a user who has delegated}
 
-As a Daedalus home user with my stake delegated to other users I would like to
-join the Cardano network so I can participate in the network.
+As a Daedalus home user with my stake delegated to other users I would like to join the Cardano network
+so I can participate in the network.
 \begin{itemize}
-\item The system be designed to provide this user segment with the ability
-      to catch up and keep up with the blockchain without having to do any local
-      network configuration.
-\item The system be designed to provide this user segment with the ability to
+\item The system must be designed to provide this user segment with the ability
```

Comment:
where did "user segment" notion come from - are user's now like oranges? Isn' t it just a "user"?

Reply:
It was copy&pasted from a protocol.tex the old version of the documentation.
Leaving this as is it.
later on there is  'Users in this segment will be defined as'...